azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/PackageMojo.java

```diff
@@ -254,9 +257,11 @@ protected void writeObjectToFile(final ObjectWriter objectWriter, final Object o
     }
 
     protected ObjectWriter getObjectWriter() {
+        final DefaultIndenter indenter = DefaultIndenter.SYSTEM_LINEFEED_INSTANCE.withLinefeed("\n");
```

Comment:
use interface, not impl class

Reply:
Changed to `DefaultPrettyPrinter.Indenter`